### PR TITLE
Fixed trap-door rotation issue.

### PR DESCRIPTION
**Pull request by SirGamer**

### CODE REVIEW

File: src/pocketmine/block/Trapdoor.php [hunk 1]
```diff
@@ -18,40 +17,29 @@
  * 
  *
 */
-
 namespace pocketmine\block;
-
```
Comment (PEMapModder) [inline src/pocketmine/block/Trapdoor.php]: what?

File: src/pocketmine/block/Trapdoor.php [hunk 2]
```diff
@@ -113,43 +100,40 @@ protected function recalculateBoundingBox(){
...
 	public function place(Item $item, Block $block, Block $target, $face, $fx, $fy, $fz, Player $player = null){
 		if(($target->isTransparent() === false or $target->getId() === self::SLAB) and $face !== 0 and $face !== 1){
-			$faces = [
-				2 => 0,
-				3 => 1,
-				4 => 2,
-				5 => 3,
-			];
+			$faces = [];
+			switch($faces){
```
Comment (PEMapModder) [inline src/pocketmine/block/Trapdoor.php]: why you switch an array?

**Comment by SirGamer** [comment]
Idk...

Maybe because I'm a nub...

**Comment by PEMapModder** [comment]
I guess there are plenty of inappropriate changes here.

**Pull request closed without merging**